contrib/inventory/rax.py

```diff
@@ -151,7 +151,10 @@
 import argparse
 import warnings
 import collections
-import ConfigParser
+try:
```

Comment:
I've just been informed we have something built in for this:

```suggestion
from ansible.module_utils.six.moves import configparser
```

https://pythonhosted.org/six/#module-six.moves

Reply:
Thanks, changed.